src/common/common.cpp

```diff
@@ -34,12 +33,67 @@ std::optional<POINT> get_mouse_pos() {
   }
 }
 
+static bool is_system_window(HWND hwnd, const char* class_name = nullptr) {
```

Comment:
Can you add a comment to explain the purpose of this method and some of the logic used internally?

Reply:
Ok